paddle/fluid/eager/auto_code_generator/final_state_generator/python_c_gen.py

```diff
@@ -328,9 +329,13 @@ def GeneratePythonCFunction(self):
             get_eager_tensor_str, parse_attributes_str, fwd_function_name,
             dygraph_function_call_str, return_str)
 
+        # Set prefix of forward_api_name to avoid conflicts
+        prefix = self.namespace.strip("::")
+        forward_api_name_prefix = "" if prefix == "" else prefix + "_"
```

Comment:
Per discussion with Yunfei and Kaihuo, "namespace" will be applied to separate different groups of APIs such as Regular API and Sparse API or String API in the future. 

To stay consistent with the implementations of Dygraph Functions and C++ API, it is suggested that we should stick to the namespace manner instead of make use of prefix/suffix.

Reply:
主要是为了生成的pybind接口冲突。string tensor与dense tensor存在同名api，目前统一生成的接口是放在`core.eager.ops`下面，需要添加prefix避免同名冲突。后续可以添加`core.eager.ops.sparse`以及`core.eager.ops.strings`模块，将sparse以及strings 的api分别迁移到这两个模块下，避免api同名冲突